### PR TITLE
Martin's fixes to the SUSE layout for labsconf 2021

### PR DESCRIPTION
**Pull request by mwilck**
Hi Cédric, 

as discussed previously, here's a PR with my improvements to our reveal.js company slides.

I've made a lot of changes, I think most of them are for the better. I payed attention to the PDF layout, too. Unfortunately I don't have time to clean up the git history. So here's all my changes except the content of my presentation.

View the slides under [on github.io](https://mwilck.github.io/presentation-template) to get an impression; [PDF pages](https://mwilck.github.io/presentation-template?print-pdf). You may also look at  [my labsconf slides](https://mwilck.github.io/labsconf2021).

Cheers
Martin

### CODE REVIEW

**Comment by mwilck** [comment]
Ping?

File: bin/publish.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+mkdir -p public
+rsync -ric --exclude=/public \
+      --include=reveal.js/dist --include=reveal.js/plugin --exclude=reveal.js/* \
+      --exclude=/.* --include=* ./ public/
+sass --update reveal.js/css/print:public/reveal.js/dist/print
+sass -E utf-8 --update css public/css
```
Comment (ddiss) [inline bin/publish.sh:9]: I realize it's just a helper script, but some error checking would probably still be worthwhile.

**Review by ddiss — APPROVED** [review]
Seems fine to me. I haven't looked closely at the css changes especially, but the rendered output looks good.

File: bin/publish.sh (hunk 1 above)
Comment (mwilck) [reply bin/publish.sh:9]: Do you have anything special in mind?

I've seen sass fail quite a few times while I was hacking on the CSS. It produces more or less understandable output for the dev to fix the issue.

Comment (ddiss) [reply bin/publish.sh:9]: nothing special - exit with non-zero if any of the commands fail?

Comment (mwilck) [reply bin/publish.sh:9]: should be fixed with latest push

File: bin/publish.sh [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #! /bin/sh
+trap 'echo "$0: error in command \"$BASH_COMMAND\"" >&2; exit 1' ERR
```
Comment (ddiss) [inline bin/publish.sh:2]: `#! /bin/sh` should be changed now that this is bash specific.

**Review by ddiss — CHANGES_REQUESTED** [review]

**Review by ddiss — APPROVED** [review]

**Comment by mwilck** [comment]
ping ...

**Comment by thesp0nge** [comment]
Can we have this PR merged in official repo?

**Comment by cbosdo** [comment]
Had to manually merge it

**Comment by mwilck** [comment]
Thanks! Sorry for the merge efforts, I suppose I missed a rebase?

**Comment by cbosdo** [comment]
Now it's properly pushed

**Comment by cbosdo** [comment]
> Thanks! Sorry for the merge efforts, I suppose I missed a rebase?

No it's my fault: I just merged another PR that was conflicting with yours

**Pull request closed without merging**